config/environments/production.rb

```diff
@@ -81,7 +81,7 @@
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
   if ENV['RAILS_LOG_TO_STDOUT'].present?
-    logger           = ActiveSupport::Logger.new(STDOUT)
+    logger           = ActiveSupport::Logger.new($stdout)
```

Comment:
Is this here for a dependency update? This doesn't on a glance seem to do with the task at hand. I might prefer in such a case just a quick note about it?

Reply:
That's right — this is one of several changes that was made in order to placate Rubocop in https://github.com/artsy/rosalind/pull/586/commits/0521c3bdb5e822bbe4aa2a3d86ed7fd1cb38d1e3

And that's a consequence of upgrading this project to a recent enough Rubocop to get the new settings.